src/ripple/protocol/impl/ErrorCodes.cpp

```diff
@@ -88,10 +87,8 @@ constexpr static ErrorInfo unorderedErrorInfos[]{
     {rpcNOT_IMPL, "notImpl", "Not implemented."},
     {rpcNOT_READY, "notReady", "Not ready to handle this request."},
     {rpcNOT_SUPPORTED, "notSupported", "Operation not supported."},
-    {rpcNO_CLOSED, "noClosed", "Closed ledger is unavailable."},
-    {rpcNO_CURRENT, "noCurrent", "Current ledger is unavailable."},
+    {rpcNOT_SYNCED, "notSynced", "Not synced to XRP network."},
```

Comment:
I would say maybe remove the word "XRP" here and just say "to the network"? Since that way the message isn't tied to any specific branding and it doesn't have to say _which_ network you're syncing to.

Reply:
Done.